peekingduck/pipeline/nodes/model/jde.py

```diff
@@ -12,83 +12,113 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-"""
-Human detection and tracking model.
-"""
+"""Human detection and tracking model."""
 
+from pathlib import Path
 from typing import Any, Dict
-from peekingduck.pipeline.nodes.node import AbstractNode
+
 from peekingduck.pipeline.nodes.model.jdev1 import jde_model
+from peekingduck.pipeline.nodes.node import AbstractNode
 
 
 class Node(AbstractNode):
-    """Initialise and use the JDE model to detect and track people from
-    image frame.
+    """Initialises and uses JDE tracking model to detect and track people from
+    the supplied image frame.
 
-    The JDE model node allows target detection and appearance embedding
-    to be learned in a shared model. In addition, the authors further
-    propose a simple and fast association method that works in conjunction
-    with the joint model. This implementation is inference only, and any
-    training code is stripped from the repository and model weights.
+    JDE is a fast and high-performance multiple-object tracker that learns the
+    object detection task and appearance embedding task simultaneously in a
+    shared neural network.
 
     Inputs:
         |img|
 
     Outputs:
         |bboxes|
 
-        |obj_tags|
-
         |bbox_labels|
 
+        |bbox_scores|
+
+        |obj_tags|
+
     Configs:
-        weights_dir (:obj:`List`):
-            Directory pointing to the model weights.
-        blob_file (:obj:`str`):
-            Mame of file to be downloaded, if weights are not found in
-            ``weights_dir``.
-        model_files (:obj:`Dict`):
-            Dictionary pointing to path of the model weights file and model
-            config file.
+        model_type (:obj:`str`): **{"576x320", "865x480", "1088x608},
+            default="576x320"**. |br|
```

Comment:
I see only 1 JDE model file `jde_864x480_uncertainty-stripped.pt` in the weights. Do all 3 options still exist then? Also:
1) It should be 864 instead of 865?
2) The last option is missing a closing double quotes

Reply:
Leftover code from trying to speed up development by running a smaller model. Removed the config docstring since it's not meant to be adjustable.